test/tests/multiapps/command_line/tests

```diff
@@ -5,20 +5,63 @@
     type = Exodiff
     input = parent.i
     exodiff = 'parent_out_sub0.e parent_out_sub1.e'
-    requirement = "The MultiApp system shall include the ability to set command line parameters for each sub application from the input file."
+    requirement = "The MultiApp system shall include the ability to set command line parameters for "
```

Comment:
Boo at the use of an implementation detail but as we have published on this name for a decade and it is a part of what makes MOOSE MOOSE, then it seems fine, e.g. it's more of a colloquial noun at this point as opposed to an implementation noun